README.md

```diff
@@ -202,31 +228,47 @@ the complete HTML document, you can use the `emogrifyBodyContent` instead:
 $bodyContent = $emogrifier->emogrifyBodyContent();
 ```
 
-#### Options
+### Migrating from `Emogrifier` to `CssInliner`
 
-There are several options that you can set on the `CssInliner` instance before
-calling the `inlineCss` method (or on the `Emogrifier` instance before calling
-the `emogrify` method):
+#### Minimal example
 
-* `->disableStyleBlocksParsing()` - By default, `CssInliner` will grab
-  all `<style>` blocks in the HTML and will apply the CSS styles as inline
-  "style" attributes to the HTML. The `<style>` blocks will then be removed
-  from the HTML. If you want to disable this functionality so that `CssInliner`
-  leaves these `<style>` blocks in the HTML and does not parse them, you should
-  use this option. If you use this option, the contents of the `<style>` blocks
-  will _not_ be applied as inline styles and any CSS you want `CssInliner` to
-  use must be passed in as described in the [Usage section](#usage) above.
-* `->disableInlineStylesParsing()` - By default, `CssInliner`
-  preserves all of the "style" attributes on tags in the HTML you pass to it.
-  However if you want to discard all existing inline styles in the HTML before
-  the CSS is applied, you should use this option.
-* `->addAllowedMediaType(string $mediaName)` - By default, `CssInliner`
-  will keep only media types `all`, `screen` and `print`. If you want to keep
-  some others, you can use this method to define them.
-* `->removeAllowedMediaType(string $mediaName)` - You can use this
-  method to remove media types that Emogrifier keeps.
-* `->addExcludedSelector(string $selector)` - Keeps elements from
-  being affected by CSS inlining.
+Old code using `Emogrifier`:
+
+```php
+$emogrifier = new Emogrifier($html);
+$html = $emogrifier->emogrify();
+```
+
+New code using `CssInliner`:
+
+```php
+$html = CssInliner::fromHtml($html)->inlineCss()->render();
+```
+
+NB: In this example, the old code removes elements with `display: none;`
+while the new code does not, as the default behaviors of the old and
+the new class differ in this regard.
+
+#### More complex example
+
+Old code using `Emogrifier`:
+
+```php
+$emogrifier = new Emogrifier($html, $css);
+$emogrifier->enableCssToHtmlMapping();
+
+$html = $emogrifier->emogrify();
+```
+
+New code using `CssInliner` and family:
+
+```php
+$domDocument = CssInliner::fromHtml($html)->inlineCss($css)->getDomDocument();
+
+HtmlPruner::fromDomDocument($domDocument)->removeInvisibleNodes(),
```

Comment:
`removeInvisibleNodes` was renamed to `removeElementsWithDisplayNone`, so this needs updating.

Reply:
Good catch! Fixed and repushed.